platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -335,6 +345,8 @@ productivity include the following:
 - Active users and resources: number of capabilities provisioned and used from the platform
 - "Net Promoter Score" (NPS) or other surveys that measure user satisfaction with a product
 - Metrics for developer productivity such as those discussed in the SPACE framework [[3]]
+- Monthly adoption rate and churn - measuring these two against each other can also help
```

Comment:
The "Active users" item above captures this measure. If you feel deprovisioning is important to measure too could we modify the above item as follows? Thanks!

"Active users and resources: number of capabilities provisioned and used or deprovisioned via the platform"

Reply:
so on the metrics side - I believe active users is different than monthly adoption rate. active users measures daily engagement and that can be amongst your current users

monthly adoption rate and churn is meant for adoption - how many customers are you gaining and losing